purchase_sale_container_deposit_inter_company/models/purchase_order.py

```diff
@@ -20,3 +20,9 @@ def _prepare_sale_order_line_data(self, purchase_line, dest_company, sale_order)
         # Copy value from PO to SO
         order_line["is_container_deposit"] = purchase_line.is_container_deposit
         return order_line
+
+    def write(self, vals):
+        # Allow update of so lines on locked sale on update of order container deposit quantity
+        if self.env.context.get("update_order_container_deposit_quantity", False):
```

Comment:
Can we have a test that check that we are able to update locked sales while we are updating order container deposit ?

Reply:
Hello @santostelmo , I updated